src/converse-chatview.js

```diff
@@ -376,10 +376,20 @@
 
                 handleChatStateMessage: function (message) {
                     if (message.get('chat_state') === _converse.COMPOSING) {
-                        this.showStatusNotification(message.get('fullname')+' '+__('is typing'));
+                        if(message.get('sender') === 'me') {
+                            this.showStatusNotification('Typing from another device');
+                        }
```

Comment:
User-facing strings should be wrapped in `__( )` so that they will get translated.